src/actuator/ActuatorParsingFAST.C

```diff
@@ -11,6 +11,8 @@
 #include <NaluParsing.h>
 #include <actuator/ActuatorParsingFAST.h>
 #include <NaluEnv.h>
+#include <yaml-cpp/node/node.h>
+#include <yaml-cpp/node/type.h>
```

Comment:
Why not 

```C++
#include <yaml-cpp/yaml.h>
```

Reply:
This was an IDE plugin's fault. Didn't notice it until now. These aren't even necessary.

Reply:
I'll clean up the unnecessary includes